cmds/webboot/webboot.go

```diff
@@ -229,10 +230,14 @@ func main() {
 	}
 	entry := getMainMenu(cacheDir)
 
-	var err error
+	// Buffer the log output, else it might overlap with the menu
+	var logBuffer bytes.Buffer
```

Comment:
do you have a way to display this output anywhere? is this a bitbucket for errors?

Reply:
Right now it's just collecting things that get written to the log. I think the only function that writes to the log is `syslinux.ParseLocalConfig()`, and all it displayed was a formatted list of the boot configurations it found.

Reply:
In the future, maybe we could pass the buffer to the `Progress` widget and have it display the output live. For now though, I was just buffering the output and storing it for a later use case.